src/components/Modal/index.js

```diff
@@ -193,53 +196,72 @@ class Modal extends React.Component {
       containerProps,
       actionBarProps,
       contentProps,
-      bottomActionBarProps
+      bottomActionBarProps,
+      displayTop
```

Comment:
Can you please add this new prop to the catalog page?

Reply:
Done